packages/web-components/src/index-rollup.ts

```diff
@@ -1,203 +1,35 @@
-export { Radio, RadioDefinition, RadioStyles, RadioTemplate } from './radio/index.js';
-export {
-  RadioGroup,
-  RadioGroupOrientation,
-  RadioGroupDefinition,
-  RadioGroupStyles,
-  RadioGroupTemplate,
-} from './radio-group/index.js';
-export {
-  Slider,
-  SliderMode,
-  SliderOrientation,
-  SliderSize,
-  SliderDefinition,
-  SliderStyles,
-  SliderTemplate,
-} from './slider/index.js';
-export {
-  Spinner,
-  SpinnerAppearance,
-  SpinnerSize,
-  SpinnerTemplate,
-  SpinnerStyles,
-  SpinnerDefinition,
-} from './spinner/index.js';
-export { Switch, SwitchLabelPosition, SwitchDefinition, SwitchStyles, SwitchTemplate } from './switch/index.js';
-export { Tab, TabTemplate, TabStyles, TabDefinition } from './tab/index.js';
-export { TabPanel, TabPanelTemplate, TabPanelStyles, TabPanelDefinition } from './tab-panel/index.js';
-export {
-  Tabs,
-  TabsAppearance,
-  TabsOrientation,
-  TabsSize,
-  TabsTemplate,
-  TabsStyles,
-  TabsDefinition,
-} from './tabs/index.js';
-export {
-  Text,
-  TextAlign,
-  TextFont,
-  TextSize,
-  TextWeight,
-  TextTemplate,
-  TextStyles,
-  TextDefinition,
-} from './text/index.js';
-export {
-  TextInput,
-  TextInputType,
-  TextInputAppearance,
-  TextInputControlSize,
-  TextInputTemplate,
-  TextInputStyles,
-  TextInputDefinition,
-} from './text-input/index.js';
-export {
-  ToggleButton,
-  ToggleButtonAppearance,
-  ToggleButtonShape,
-  ToggleButtonSize,
-  ToggleButtonTemplate,
-  ToggleButtonStyles,
-  ToggleButtonDefinition,
-} from './toggle-button/index.js';
+import './accordion-item/define.js';
+import './accordion/define.js';
+import './anchor-button/define.js';
+import './avatar/define.js';
+import './badge/define.js';
+import './button/define.js';
+import './checkbox/define.js';
+import './compound-button/define.js';
+import './counter-badge/define.js';
+import './dialog/define.js';
+import './divider/define.js';
+import './label/define.js';
+import './menu-button/define.js';
+import './menu-item/define.js';
+import './menu-list/define.js';
+import './menu/define.js';
+import './progress-bar/define.js';
+import './radio-group/define.js';
+import './radio/define.js';
+import './slider/define.js';
+import './spinner/define.js';
+import './switch/define.js';
+import './tab-panel/define.js';
+import './tab/define.js';
+import './tabs/define.js';
+import './text-input/define.js';
+import './text/define.js';
+import './toggle-button/define.js';
 
-export { getDirection } from './utils/direction.js';
-export { display } from './utils/display.js';
-export {
-  forcedColorsStylesheetBehavior,
-  darkModeStylesheetBehavior,
-  lightModeStylesheetBehavior,
-} from './utils/behaviors/match-media-stylesheet-behavior.js';
+import { webDarkTheme, webLightTheme } from '@fluentui/tokens';
+import { setTheme } from './theme/set-theme.js';
 
-export { FluentDesignSystem } from './fluent-design-system.js';
-export { setTheme, setThemeFor } from './theme/index.js';
+setTheme(webLightTheme);
```

Comment:
If we set this and they want a different theme, will we have a flash of styled => styled content?

Reply:
We may want to remove this `setTheme()` call entirely, and communicate that a theme needs to be explicitly set.

The components won't have any reasonable appearance unless tokens are present.